client/components/ReviewRating/ReviewList/ReviewBody.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import React, { useState, useEffect } from 'react';
+import ReactModal from 'react-modal';
```

Comment:
You shouldn't be using a library for your modal. For this project, you'll need to build it yourself. I believe the Related Products also uses a modal, so you guys can ideally reuse that feature :)

Reply:
I will change it